cli/src/processSourceFiles.ts

```diff
@@ -85,26 +102,54 @@ const findJSFilesRecursively = (
   }
   visitedPaths.add(realPath);
 
-  const files = fs.readdirSync(dirPath);
+  const files = fs.readdirSync(realPath);
   const jsFiles = files.filter((file) => file.endsWith('.js'));
 
   // Process JS files in current directory
   for (const jsFile of jsFiles) {
-    const mapFile = `${jsFile}.map`;
-    const jsFilePath = path.join(dirPath, jsFile);
-    const mapFilePath = path.join(dirPath, mapFile);
+    const jsFilePath = path.join(realPath, jsFile);
+
+    // Read the JS file to extract the sourceMappingURL
+    try {
+      const jsContent = fs.readFileSync(jsFilePath, 'utf-8');
+      const sourceMapUrl = extractSourceMapUrl(jsContent);
```

Comment:
Should we stil fallback to `.map` if the `sourceMappingURL` comment does not exist? I'm not sure if there are tools out there that do not add that comment?